apollo-router/src/axum_http_server_factory.rs

```diff
@@ -91,22 +91,15 @@ impl HttpServerFactory for AxumHttpServerFactory {
                 .clone()
                 .map(|cors_configuration| cors_configuration.into_layer())
                 .unwrap_or_else(|| Cors::builder().build().into_layer());
-
+            let graphql_endpoint = if configuration.server.endpoint.ends_with("/*") {
+                // Needed for axum
+                format!("{}router_extra_path", configuration.server.endpoint)
```

Comment:
Can you explain in the comment what `router_extra_path` is?

Reply:
Added the link to the axum docs